lib/rubygems/commands/sign_command.rb

```diff
@@ -21,7 +21,7 @@ module Sigstore
 require "rubygems/sigstore/config"
 require "rubygems/sigstore/crypto"
 require "rubygems/sigstore/fulcio_api"
```

Comment:
If this change makes sense for Rekor, we can follow it up with a similar refactor of Fulcio code.